web/src/routes/~_dashboard/~file/HelpDialog.tsx

```diff
@@ -0,0 +1,50 @@
+//
+// This source file is part of the Stanford Biodesign Digital Health RadGPT open-source project
+//
+// SPDX-FileCopyrightText: 2024 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import {
+  Card,
+  CardHeader,
+  CardTitle,
+} from "@stanfordspezi/spezi-web-design-system";
+import { cn } from "@stanfordspezi/spezi-web-design-system/utils/className";
+import { type ReactNode } from "react";
+
+interface LegendItemProps {
+  color: string;
+  content: ReactNode;
+}
+
+export const LegendItem = ({ color, content }: LegendItemProps) => (
+  <li className="flex items-center gap-2 text-sm">
+    <div className={cn("size-3 shrink-0 rounded-full", color)} />
+    <span>{content}</span>
+  </li>
+);
+
+export const Legend = () => (
+  <ul className="flex flex-wrap items-center gap-x-5 gap-y-1 px-5">
+    <LegendItem color="bg-blue-700" content="Identified Medical Concept" />
+    <LegendItem color="bg-green-300" content="Medical Observation" />
+    <LegendItem
+      color="bg-yellow-300"
+      content="Location of Medical Observation"
+    />
```

Comment:
```suggestion
  <ul className="flex flex-wrap items-center gap-x-5 gap-y-1 px-5">
    <LegendItem color="bg-green-300" content="Medical Observation" />
    <LegendItem
      color="bg-yellow-300"
      content="Anatomic Location of Medical Observation"
    />
```

That came from today's meeting

Reply:
Is it purposefully without blue "Identified Medical Concept"?

Reply:
[Update legend items](https://github.com/StanfordBDHG/RadGPT/pull/89/commits/23d4ab774384893c8012ee3983b5bd49db0eba71)